parsers/ajenti.py

```diff
@@ -0,0 +1,156 @@
+# Initial PR https://github.com/tmrowco/electricitymap-contrib/pull/2456
+# Discussion thread https://github.com/tmrowco/electricitymap-contrib/issues/636
+# A promotion webpage for King's Island energy production is here : https://www.hydro.com.au/clean-energy/hybrid-energy-solutions/success-stories/king-island
+# As of 09/2020, it embeds with <iframe> the URI https://data.ajenti.com.au/KIREIP/index.html
+# About the data, the feed we get seems to be counters with a 2 seconds interval.
+# That means that if we fetch these counters every 15 minutes, we only are reading "instantaneous" metters that could differ from the total quantity of energies at play. To get the very exact data, we would need to have a parser running constanty to collect those 2-sec interval counters.
+
+import asyncio
+import json
+import logging
+import arrow
+from signalr import Connection
+from requests import Session
+
+ZONE_PARAMS = {
+    'AUS-TAS-KI': {
+        "hub": "TagHub",
+        "method": "Dashboard",
+        "tz": "Australia/Currie",
+        "source": "https://www.hydro.com.au/clean-energy/hybrid-energy-solutions/success-stories/king-island" #Iframe: https://data.ajenti.com.au/KIREIP/index.html
+    },
+    # Flinders Island
+    # https://github.com/tmrowco/electricitymap-contrib/issues/2533
+    # https://en.wikipedia.org/wiki/Flinders_Island
+    'AUS-TAS-FI': {
+        "hub": "flindershub",
+        "method": "SendDashboard",
+        "tz": "Australia/Hobart",
+        "source": "https://www.hydro.com.au/clean-energy/hybrid-energy-solutions/success-stories/flinders-island"
+    },
+    # Rottnest Island
+    # https://github.com/tmrowco/electricitymap-contrib/issues/2534
+    # https://en.wikipedia.org/wiki/Rottnest_Island
+    'AUS-RI': {
```

Comment:
This should be updated to make the parser work :)
```suggestion
    'AUS-WA-RI': {
```

Reply:
Good catch! Thanks